components/script/dom/bindings/codegen/CodegenRust.py

```diff
@@ -1164,6 +1164,9 @@ def __init__(self, argument, index, args, argc, descriptorProvider,
 
         if not argument.variadic:
             if argument.optional:
+                # Treat undefined arguments as missing
+                undefined_condition = string.Template("!${args}.get(${index}).is_undefined()").substitute(replacer)
```

Comment:
Wait a sec - Ms2ger's commit [here](https://github.com/servo/rust-mozjs/commit/288af44d2da5eaa2d28df9af7577058eadad10e9) made a better version of the `.get()` method. Do you still think I should use `.index()` instead?